### ISSUE TITLE
feature request: Batch insert rows

### ISSUE THREAD

**Issue by MarkRudChristensen** [ultorg/public_issues#31]
We often have the need to insert for instance 100 rows in a table. Would it be possible to add a feature that allowed adding these 100 rows in on go? 

Solutions could be to upload a CSV-file with the 100 rows. Or, as we normally prepare the data in Excel, another solution would be to copy/paste from Excel.

**Comment by eirikbakke** [comment]
Paste from Excel would make a lot of sense, for inserting rows in Data Editing Mode. The user would have to line up the right columns prior to the copy/paste, but I suppose that's a common Excel workflow.

Alternatively, or additionally, there could be a more explicit bulk insert operation that lets rows from one table (or perspective) be inserted into another table, perhaps matching columns by column label.

I'll have to think about the design a bit, and see which approach to implement first.

Some warning will have to be shown if data types don't match. (E.g. tried to paste text into a date column.) Auto-incrementing keys (mentioned in https://github.com/ultorg/public_issues/issues/20 ) would also be helpful in combination with the bulk insert feature.

**Comment by MarkRudChristensen** [comment]
I am not sure the alternative solution would be relevant for us in this particular case where a non-tech user needs to insert for instance 100 new customers prepared in Excel. If you wish I could show you our current solution (where this is possible).

**Comment by eirikbakke** [comment]
Yeah, if the data is already in Excel it makes sense to copy and paste. There will seldom be millions of rows in these cases anyway.

> If you wish I could show you our current solution (where this is possible).

Yes, would love to see! (Here or by email ebakke@ultorg.com )

**Comment by eirikbakke** [comment]
Batch insert is now supported in Ultorg 1.4.0!

I ended up taking the paste-from Excel approach, where Ctrl+V inserts rows from the clipboard starting from the column that currently has the cell cursor in it:

![bulkPasteInsert](https://github.com/ultorg/public_issues/assets/886243/bdecaed0-0c1e-4d24-a18c-5b8a5b6539e1)

If the data types of the pasted columns don't match up, there will be a friendly error tooltip.

(See more details in the [Release Notes](https://www.ultorg.com/releasenotes/).)
